### ISSUE TITLE
Batch updates are broken since version 2.17.0

### ISSUE THREAD

**Issue by teyeheimans** [tailflow/laravel-orion#192]
I have noticed that since version 2.17.0 batch updates are not working like they used to do. 

I have a test which creates 2 database records. I then test if updates works with a test like this:

```
$this->patch('/api/media/batch', [
                'resources' => [
                    1 => [
                        "valid" => 1,
                    ],
                    2 => [
                        "valid" => 0,
                    ],
                ],
            ])            
            ->assertOk();
```

After that I do some database assertions to check if the correct actions are taken. 

In my media controller, I overwrite the fetch query where I only return media which are not validated yet:

```
    /**
     * Builds Eloquent query for fetching entity(-ies).
     *
     * @param \Orion\Http\Requests\Request $request
     * @param array                        $requestedRelations
     *
     * @return Builder
     * @throws \Psr\Container\ContainerExceptionInterface
     * @throws \Psr\Container\NotFoundExceptionInterface
     */
    protected function buildFetchQuery(Request $request, array $requestedRelations): Builder
    {
        $builder = parent::buildFetchQuery($request, $requestedRelations);

        $builder->whereRaw('NOT JSON_CONTAINS(custom_properties, "{\"checked\":true}")'); 

        return $builder;
    }
```

The old way In version 2.16.0  the response of the patch was a HTTP 200. Since version 2.17.0, I receive an HTTP 404 with the error:

```
{
  "message": "No query results for model [App\Models\Media].",
  "exception": "Symfony\Component\HttpKernel\Exception\NotFoundHttpException",
  "file": "<path>/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php",
  "line": 385,
  "trace": []
}  
```

**Comment by GautierDele** [comment]
hello @teyeheimans, is this still an issue ?

Thanks,

**Comment by teyeheimans** [comment]
Hi @GautierDele, yes, this issue still exists.

**Comment by GautierDele** [comment]
Ok, i'll have a look in the next days :)

**Comment by teyeheimans** [comment]
Awesome, thanks. I've also taken a look in the code. I see that the model is now refreshed after the update. However, because of the `buildFetchQuery` change, it will not match anymore after the update, and thus it will not find the mode (hence the NotFound exception).

**Comment by alexzarbn** [comment]
Hi @teyeheimans,

Should be fixed in [v2.17.5](https://github.com/tailflow/laravel-orion/releases/tag/2.17.5)

**Comment by teyeheimans** [comment]
Hi @alexzarbn , Thanks, I can confirm the test now passes.
